NotifyTests/UnitTests/Constants.cs

```diff
@@ -83,6 +84,29 @@ public static class Constants
             }
         }
 
+        public static String fakeReceivedTextListResponseJson { get {
+                return @"{ ""received_text_messages"": [
+                        { 
+                            ""user_number"": ""447700900111"",
+                            ""created_at"": ""2017-11-02T15:07:57.197546Z"",
+                            ""service_id"": ""a5149c32-f03b-4711-af49-ad6993797d45"",
+                            ""id"": ""342786aa-23ce-4695-9aad-7f79e68ee29a"",
+                            ""notify_number"": ""testing"",
+                            ""content"": ""Hello""
+                        },
+                        { 
+                            ""user_number"": ""447700900111"",
+                            ""created_at"": ""2017-11-02T15:07:57.197546Z"",
+                            ""service_id"": ""a5149c32-f03b-4711-af49-ad6993797d45"",
+                            ""id"": ""342786aa-23ce-4695-9aad-7f79e68ee29a"",
+                            ""notify_number"": ""testing"",
+                            ""content"": ""Hello again""
+                        }
```

Comment:
tabbing has got out of sync

Reply:
Looks fine on mine, it's in line with the other constants